File: Patrones/Estructurales/Decorator/Java/src/Leche.java
public class Leche extends DecoradorBebida {
    public Leche(Bebida bebida) {
        super(bebida);
    }

    @Override
    public String getDescripcion() {
        return bebidaDecorada.getDescripcion() + ", con Leche";
    }

    @Override
    public double getCosto() {
        return bebidaDecorada.getCosto() + 10.0;
    }
}

File: Patrones/Estructurales/Decorator/Python/Leche.py
from DecoradorBebida import DecoradorBebida

# Clase concreta Leche que extiende DecoradorBebida
class Leche(DecoradorBebida):
    
    def get_descripcion(self):
        return self._bebida_decorada.get_descripcion() + ", con Leche"

    def get_costo(self):
        return self._bebida_decorada.get_costo() + 10.0
